src/main/java/hudson/plugins/jacoco/JacocoBuildAction.java

```diff
@@ -297,12 +293,12 @@ public JacocoBuildAction getPreviousResult() {
 	 * @throws IOException
 	 *      if failed to parse the file.
 	 */
-	public static JacocoBuildAction load(AbstractBuild<?,?> owner, Rule rule, JacocoHealthReportThresholds thresholds, BuildListener listener, JacocoReportDir layout, String[] includes, String[] excludes) throws IOException {
+	public static JacocoBuildAction load(Run<?,?> owner, JacocoHealthReportThresholds thresholds, TaskListener listener, JacocoReportDir layout, String[] includes, String[] excludes) throws IOException {
```

Comment:
It seems it's not used outside this plugin. It's an acceptable change IMO. Adding the `Utils.isOverridden` dance makes the code dirty, I would try to avoid it when possible.